pact-jvm-consumer-groovy/src/main/groovy/au/com/dius/pact/consumer/groovy/BaseBuilder.groovy

```diff
@@ -4,16 +4,20 @@ package au.com.dius.pact.consumer.groovy
  * Base class for builders
  */
 class BaseBuilder extends Matchers {
-    public static final List COMPACT_MIME_TYPES = ['application/x-thrift+json']
+  public static final List COMPACT_MIME_TYPES = ['application/x-thrift+json']
 
-    def call(Closure closure) {
-        build(closure)
-    }
+  def call(Closure closure) {
+    build(closure)
+  }
 
-    def build(Closure closure) {
-        closure.delegate = this
-        closure.resolveStrategy = Closure.DELEGATE_FIRST
-        closure.call()
-    }
+  def build(Closure closure) {
+    callWithThisAsDelegate(closure)
+  }
+
+  def callWithThisAsDelegate(Closure closure) {
```

Comment:
I don't understand the need for this method. `call` invokes `build` which in turn invokes `callWithThisAsDelegate`.